crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -131,9 +141,7 @@ impl HintProcessorLogic for CairoHintProcessor<'_> {
         _reference_ids: &HashMap<String, usize>,
         _references: &[HintReference],
     ) -> Result<Box<dyn Any>, VirtualMachineError> {
-        Ok(Box::new(
-            self.original_cairo_hint_processor.string_to_hint[hint_code].clone(),
-        ))
```

Comment:
Why it had to be changed?

Reply:
I guess `SyscallHintProcessor` does not have this field?

Reply:
The field was private, we need to store a copy of it